packages/suite/src/utils/suite/device.ts

```diff
@@ -132,6 +132,16 @@ export const isSelectedDevice = (selected?: TrezorDevice | Device, device?: Trez
 
 export const isDeviceInBootloader = (device?: KnownDevice) => !!device?.features.bootloader_mode;
 
+export const isBitcoinOnly = (device: TrezorDevice | Device) => {
```

Comment:
Perhaps it's better to create a general util that would accept a `device` and an array or just one `capability` and return `true` if all passed capabilities are supported? 🤓  

Or did I misunderstand and it checks if that device already has the btc-only FW?

Reply:
This util specifically checks if Bitcoin-only firmware is installed. The way it does so (using features) is not very intuitive, so I would prefer keeping this as a separate function rather than generalize it.

Reply:
nit: `isDeviceBitcoinOnly`

Reply:
renamed in b22132bd18adf47a5e711d50663f1d79eca01cca